shared/js/smart-screen/simple_key_navigation.js

```diff
@@ -30,7 +32,29 @@
   };
 
   proto.stop = function skn_stop() {
-    this.target.removeEventListener('keydown', this);
+    if (!this._started) {
```

Comment:
Since we are going to rename the "stop" method above, I recommend renaming this private variable `_started` as well.